src/users/views.py

```diff
@@ -75,17 +82,40 @@ def post(self, request):
             last_name=request.data.get('last_name', ''),
         )
 
+        check = re.search(r'[\w.]+\@(g.)?ucla.edu', new_user.email)
+        if check is None:
+            raise Exception("Invalid Email")
```

Comment:
use `from rest_framework.exceptions import ValidationError` and `ValidationError`.  This will throw a 400 code to the user.  And maybe say `'Invalid UCLA email'`.